resume-buddy/src/main/webapp/scripts/resume-review-script.js

```diff
@@ -91,7 +91,8 @@ async function getRevieweeResume() {
         metaData: {
           fileName: "revieweeResume.pdf"
         }
-      }, {embedMode : "IN-LINE"});
+        //TODO (sesexton): Add embed mode correctly
```

Comment:
What happened with the INLINE embed mode?

Reply:
I was getting a error because it says It wasn't added properly, but It should be added now. I also disabled the annotation tools and such untill we are able to save the annotations.